noetic/Dockerfile

```diff
@@ -71,9 +71,11 @@ RUN /bin/sh -c 'echo "deb http://packages.ros.org/ros/ubuntu ${RELEASE} main" >
     ros-${DIST}-kdl-parser-py \
     ros-${DIST}-key-teleop \
     ros-${DIST}-move-base \
+    ros-${DIST}-moveit* \
```

Comment:
I'm assuming that the `*` is a wildcard and will install all packages that start with `moveit`.  If so, I think we should be more selective.  It looks like there are many packages that would meet this pattern:
```
bsb@8f03a1eebd5f:~$ apt list | grep noetic-moveit

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

ros-noetic-moveit-chomp-optimizer-adapter-dbgsym/focal 1.1.8-1focal.20220212.170918 amd64
ros-noetic-moveit-chomp-optimizer-adapter/focal 1.1.8-1focal.20220212.170918 amd64
ros-noetic-moveit-commander/focal 1.1.8-1focal.20220212.175350 amd64
ros-noetic-moveit-core-dbgsym/focal 1.1.8-1focal.20220212.164344 amd64
ros-noetic-moveit-core/focal 1.1.8-1focal.20220212.164344 amd64
ros-noetic-moveit-fake-controller-manager-dbgsym/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-fake-controller-manager/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-kinematics-dbgsym/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-kinematics/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-msgs/focal 0.11.2-1focal.20220107.005220 amd64
ros-noetic-moveit-opw-kinematics-plugin-dbgsym/focal 0.4.0-3focal.20220212.172939 amd64
ros-noetic-moveit-opw-kinematics-plugin/focal 0.4.0-3focal.20220212.172939 amd64
ros-noetic-moveit-planners-chomp-dbgsym/focal 1.1.8-1focal.20220212.175350 amd64
ros-noetic-moveit-planners-chomp/focal 1.1.8-1focal.20220212.175350 amd64
ros-noetic-moveit-planners-ompl-dbgsym/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-planners-ompl/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-planners/focal 1.1.8-1focal.20220212.180803 amd64
ros-noetic-moveit-plugins/focal 1.1.8-1focal.20220212.173224 amd64
ros-noetic-moveit-python/focal 0.4.3-1focal.20220114.185413 amd64
ros-noetic-moveit-resources-fanuc-description/focal 0.8.2-1focal.20211118.225917 amd64
ros-noetic-moveit-resources-fanuc-moveit-config/focal 0.8.2-1focal.20220212.161046 amd64
ros-noetic-moveit-resources-panda-description/focal 0.8.2-1focal.20211118.225920 amd64
ros-noetic-moveit-resources-panda-moveit-config/focal 0.8.2-1focal.20220212.163709 amd64
ros-noetic-moveit-resources-pr2-description/focal 0.8.2-1focal.20211118.225922 amd64
ros-noetic-moveit-resources-prbt-ikfast-manipulator-plugin-dbgsym/focal 0.8.2-1focal.20220212.170358 amd64
ros-noetic-moveit-resources-prbt-ikfast-manipulator-plugin/focal 0.8.2-1focal.20220212.170358 amd64
ros-noetic-moveit-resources-prbt-moveit-config/focal 0.8.2-1focal.20220212.170702 amd64
ros-noetic-moveit-resources-prbt-pg70-support/focal 0.8.2-1focal.20220212.170727 amd64
ros-noetic-moveit-resources-prbt-support/focal 0.8.2-1focal.20220212.161327 amd64
ros-noetic-moveit-resources/focal 0.8.2-1focal.20220212.163739 amd64
ros-noetic-moveit-ros-benchmarks-dbgsym/focal 1.1.8-1focal.20220212.173412 amd64
ros-noetic-moveit-ros-benchmarks/focal 1.1.8-1focal.20220212.173412 amd64
ros-noetic-moveit-ros-control-interface-dbgsym/focal 1.1.8-1focal.20220212.170843 amd64
ros-noetic-moveit-ros-control-interface/focal 1.1.8-1focal.20220212.170843 amd64
ros-noetic-moveit-ros-manipulation-dbgsym/focal 1.1.8-1focal.20220212.174219 amd64
ros-noetic-moveit-ros-manipulation/focal 1.1.8-1focal.20220212.174219 amd64
ros-noetic-moveit-ros-move-group-dbgsym/focal 1.1.8-1focal.20220212.173452 amd64
ros-noetic-moveit-ros-move-group/focal 1.1.8-1focal.20220212.173452 amd64
ros-noetic-moveit-ros-occupancy-map-monitor-dbgsym/focal 1.1.8-1focal.20220212.170355 amd64
ros-noetic-moveit-ros-occupancy-map-monitor/focal 1.1.8-1focal.20220212.170355 amd64
ros-noetic-moveit-ros-perception-dbgsym/focal 1.1.8-1focal.20220212.172941 amd64
ros-noetic-moveit-ros-perception/focal 1.1.8-1focal.20220212.172941 amd64
ros-noetic-moveit-ros-planning-dbgsym/focal 1.1.8-1focal.20220212.171723 amd64
ros-noetic-moveit-ros-planning-interface-dbgsym/focal 1.1.8-1focal.20220212.174701 amd64
ros-noetic-moveit-ros-planning-interface/focal 1.1.8-1focal.20220212.174701 amd64
ros-noetic-moveit-ros-planning/focal 1.1.8-1focal.20220212.171723 amd64
ros-noetic-moveit-ros-robot-interaction-dbgsym/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-ros-robot-interaction/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-ros-visualization-dbgsym/focal 1.1.8-1focal.20220212.175351 amd64
ros-noetic-moveit-ros-visualization/focal 1.1.8-1focal.20220212.175351 amd64
ros-noetic-moveit-ros-warehouse-dbgsym/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-ros-warehouse/focal 1.1.8-1focal.20220212.172940 amd64
ros-noetic-moveit-ros/focal 1.1.8-1focal.20220212.180559 amd64
ros-noetic-moveit-runtime/focal 1.1.8-1focal.20220212.180828 amd64
ros-noetic-moveit-servo-dbgsym/focal 1.1.8-1focal.20220212.175351 amd64
ros-noetic-moveit-servo/focal 1.1.8-1focal.20220212.175351 amd64
ros-noetic-moveit-setup-assistant-dbgsym/focal 1.1.8-1focal.20220212.180558 amd64
ros-noetic-moveit-setup-assistant/focal 1.1.8-1focal.20220212.180558 amd64
ros-noetic-moveit-sim-controller-dbgsym/focal 0.3.0-1focal.20220212.172943 amd64
ros-noetic-moveit-sim-controller/focal 0.3.0-1focal.20220212.172943 amd64
ros-noetic-moveit-simple-controller-manager-dbgsym/focal 1.1.8-1focal.20220212.170354 amd64
ros-noetic-moveit-simple-controller-manager/focal 1.1.8-1focal.20220212.170354 amd64
ros-noetic-moveit-visual-tools-dbgsym/focal 3.6.0-1focal.20220212.172943 amd64
ros-noetic-moveit-visual-tools/focal 3.6.0-1focal.20220212.172943 amd64
ros-noetic-moveit/focal 1.1.8-1focal.20220212.181457 amd64
```

Reply:
@bsb808 Probably should have set as draft. I only considered it this morning in terms of testing the other PRs.

Reply:
ros-noetic-moveit alone is giving me errors based on missing some of the rviz functionality.  ros-noetic-moveit-ros-visualization should be installed as part of the metapackage, but it looks like that's what might be missing.  ros-noetic-moveit-visual-tools could also be the problem.